web/src/main/java/org/wso2/testgrid/web/utils/Constants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.testgrid.web.utils;
+
+/**
+ * This class will handle all the constants in the web-app.
+ */
+public class Constants {
+    public static final String JENKINS_HOME = System.getenv("JENKINS_HOME");
+    //Base64 encoded basic-auth key of the Jenkins user.
+    public static final String JENKINS_USER_AUTH_KEY = System.getenv("JENKINS_USER_AUTH_KEY");
+    public static final String BLUE_OCEAN_URI = "/blue/organization/jenkins";
+}
```

Comment:
Missing newline

Reply:
If its the new line at the EOF you refers, then it exists as mentioned in earlier comment.